src/exo/LoopIR_scheduling.py

```diff
@@ -18,6 +18,7 @@
 from .new_eff import (
     SchedulingError,
     Check_ReorderStmts,
+    Check_ParallelizeLoop,
```

Comment:
This is unnecessary, right?

Reply:
Do you use Check_ParallelizeLoop in LoopIR_scheduling.py? If so we shouldn't right?